backend/app/celery/tasks/evaluation_fast.py

```
@@ -0,0 +1,51 @@
+from app.celery.celery_app import celery_app
+from app.celery.tasks.job_execution import _run_with_otel_parent, _set_trace
+from app.celery.utils import gevent_timeout
+from app.core.config import settings
+
+logger = logging.getLogger(__name__)
+
+
+@celery_app.task(bind=True, queue="evaluations", priority=6)
+@gevent_timeout(settings.CELERY_TASK_SOFT_TIME_LIMIT, "run_evaluation_fast")
+def run_evaluation_fast(
+    self, eval_run_id: int, trace_id: str = "N/A", **kwargs
```

Ruff (0.15.15)

[warning] Unused function argument: `kwargs`

(ARG001)